packages/edit-site/src/components/global-styles/global-styles-provider.js

```diff
@@ -62,40 +69,74 @@ function useGlobalStylesUserConfig() {
 						_globalStylesId
 				  )
 				: undefined;
+			const _associatedStyleId = record
```

Comment:
As I commented in the PHP PR as well, we need to explain in the comments what associated styles are.